onnxruntime/test/providers/cpu/math/einsum_test.cc

```diff
@@ -352,13 +352,16 @@ TEST(Einsum, ExplicitEinsumAsDiagonalOpWithTranspose) {
   test.Run();
 }
 
+// ROCm doesn't support double
+#ifdef USE_CUDA
```

Comment:
But this will lose test coverage for CPU right ?

Reply:
good catch